smalltalksrc/VMMaker/CogMethod.class.st

```diff
@@ -86,35 +93,32 @@ CogMethod class >> instVarNamesAndTypesForTranslationDo: aBinaryBlock [
 	"enumerate aBinaryBlock with the names and C type strings for the
 	 inst vars to include in a CogMethod or CogBlockMethod struct."
 
-	self allInstVarNames do:
-		[:ivn|
-		"Notionally objectHeader is in a union with homeOffset and startpc but
+	self filteredInstVarNames do: [ :ivn | "Notionally objectHeader is in a union with homeOffset and startpc but
 		 we don't have any convenient support for unions.  So hack, hack, hack, hack."
```

Comment:
What means `union` in this comment? And why we don't support it? :P

Reply:
I'm not sure but maybe this comment should be in `CogMethod >> filteredInstaVarNames` now 🤔